sktime/transformations/series/compose.py

```diff
@@ -80,6 +80,7 @@ def __init__(self, transformer, passthrough=False):
         self.passthrough = passthrough
         self._is_fitted = False
         super(OptionalPassthrough, self).__init__()
+        self.clone_tags(transformer)
```

Comment:
what is this? Should this not be in a different PR?

Reply:
I added it here to avoid spamming the repo with tiny PRs as CI/CD seems to be slow.

Reply:
ok, fine